kits/agp1/test/agp1.js

```diff
@@ -0,0 +1,256 @@
+const getBlockNumber = require('@aragon/test-helpers/blockNumber')(web3)
+const getBlock = require('@aragon/test-helpers/block')(web3)
+const getBalance = require('@aragon/test-helpers/balance')(web3)
+const timeTravel = require('@aragon/test-helpers/timeTravel')(web3)
+const { assertRevert } = require('@aragon/test-helpers/assertThrow')
+const namehash = require('eth-ens-namehash').hash
+const keccak256 = require('js-sha3').keccak_256
+
+const { encodeCallScript, EMPTY_SCRIPT } = require('@aragon/test-helpers/evmScript')
+const deployAgp1 = require('../scripts/deploy_agp1.js')
+
+const Finance = artifacts.require('Finance')
+const Vault = artifacts.require('Vault')
+const Voting = artifacts.require('Voting')
+
+const getContract = name => artifacts.require(name)
+
+const pct16 = x => new web3.BigNumber(x).times(new web3.BigNumber(10).toPower(16))
+const getEventResult = (receipt, event, param) => receipt.logs.filter(l => l.event == event)[0].args[param]
+const createdVoteId = receipt => getEventResult(receipt, 'StartVote', 'voteId')
+const networks = require("@aragon/os/truffle-config").networks
+const getNetwork = require('../../../helpers/networks.js')
+
+contract('AGP-1 Kit', accounts => {
+    const ETH = '0x0'
+    const NO_ADDRESS = '0x0000000000000000000000000000000000000000'
+    const NEEDED_SUPPORT = pct16(50)
+    //const NEEDED_SUPPORT_META_TRACK = new web3.BigNumber(666666666666666667)
+    const MINIMUM_ACCEPTANCE_QUORUM = 0
+    const VOTING_TIME = 48 * 3600 // 48h
+    let daoAddress, tokenAddress, finance, vault, voting, metaTrackVoting
+
+    const owner = accounts[0]
+    const holder16 = accounts[1]
+    const holder33 = accounts[2]
+    const holder51 = accounts[3]
+    const nonHolder = accounts[4]
+
+    before(async () => {
+        const networkName = (await getNetwork(networks)).name
+        if (networkName == 'devnet' || networkName == 'rpc') {
+            // transfer some ETH to other accounts
+            await web3.eth.sendTransaction({ from: owner, to: holder16, value: web3.toWei(1, 'ether') })
```

Comment:
Is this only needed for when we are testing using geth? AFAIK, ganache funds the first 10 addresses with 100 eth.

Reply:
Yes, I used this to test it with our docker image, but I think it's enough to test with ganache, so I'll remove it.

Reply:
I would maintain the real docker tests if possible actually

Reply:
Ok, but I'll put it in a separate file because `assertRevert` and `timeTravel` were not working there for me.